docs/content/app-protect/configuration.md

```diff
@@ -1,14 +1,13 @@
 ---
 title: Configuration
 
-description: "This document describes how to configure the NGINX App Protect module"
+description: "This document describes how to configure the NGINX App Protect module."
 weight: 1900
 doctypes: [""]
 toc: true
 docs: "DOCS-578"
 ---
 
-This document describes how to configure the NGINX App Protect module
 > Check out the complete [NGINX Ingress Controller with App Protect example resources on GitHub](https://github.com/nginxinc/kubernetes-ingress/tree/v2.3.0/examples/appprotect).
```

Comment:
This should also include the Custom Resource sample: https://github.com/nginxinc/kubernetes-ingress/tree/main/examples/custom-resources/waf

Reply:
Still open